resolvers/webpack/index.js

```diff
@@ -265,7 +266,7 @@ function findExternal(source, externals, context) {
 
   // else, vanilla object
   for (var key in externals) {
-    if (!externals.hasOwnProperty(key)) continue
+    if (!has(externals, key)) continue
```

Comment:
Looks ready to :shipit:, other than this, though. 😄 Thanks to both of you for working through @jfmengels' feedback, too!